apps/web/components/shared/project-card-variants.tsx

```
@@ -1,384 +1,384 @@
+			</div>
 
-            <button
-                onClick={() => setShowImpact(true)}
-                className="w-full bg-blue-600 text-white py-3 rounded-lg font-medium mb-6 flex items-center justify-center gap-2"
-            >
-                Support this project
-            </button>
+			<button
+				onClick={() => setShowImpact(true)}
+				className="w-full bg-blue-600 text-white py-3 rounded-lg font-medium mb-6 flex items-center justify-center gap-2"
+			>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)